src/operator/nn/fully_connected-inl.h

```diff
@@ -1,4 +1,4 @@
-/*
+  /*
```

Comment:
indent

Reply:
thanks, often happens to me due to vim bindings.